packages/memory-inspector/src/browser/memory-inspector-frontend-contribution.ts

```diff
@@ -0,0 +1,301 @@
+        );
+        registerMenuActions(
+            RegisterTableWidget.CONTEXT_MENU,
+            RegisterSetVariableCommand,
+        );
+    }
+
+    registerToolbarItems(toolbarRegistry: TabBarToolbarRegistry): void {
+        toolbarRegistry.registerItem({
+            id: CreateNewMemoryViewCommand.id,
+            command: CreateNewMemoryViewCommand.id,
+            tooltip: CreateNewMemoryViewCommand.label,
+            priority: -2,
+        });
+        toolbarRegistry.registerItem({
+            id: CreateNewRegisterViewCommand.id,
+            command: CreateNewRegisterViewCommand.id,
+            tooltip: CreateNewRegisterViewCommand.label,
+            priority: -1,
+        });
+        toolbarRegistry.registerItem({
+            id: ToggleDiffSelectWidgetVisibilityCommand.id,
+            command: ToggleDiffSelectWidgetVisibilityCommand.id,
+            tooltip: nls.localize('theia/memory-inspector/toggleComparisonWidgetVisibility', 'Toggle Comparison Widget Visibility'),
+            priority: -3,
+            onDidChange: this.memoryWidgetManager.onChanged,
+        });
+    }
+
+    registerColors(colorRegistry: ColorRegistry): void {
+        colorRegistry.register(
+            {
+                id: 'diffEditor.removedTextBackground.light',
+                defaults: {
+                    dark: Color.transparent('diffEditor.removedTextBackground', ONE_HALF_OPACITY),
+                    light: Color.transparent('diffEditor.removedTextBackground', ONE_HALF_OPACITY),
+                },
+                description: 'An even lighter version of the diff editor colors for contexts when contrast is an issue.',
+            },
+            {
+                id: 'diffEditor.insertedTextBackground.light',
+                defaults: {
+                    dark: Color.transparent('diffEditor.insertedTextBackground', ONE_HALF_OPACITY),
+                    light: Color.transparent('diffEditor.insertedTextBackground', ONE_HALF_OPACITY),
+                },
+                description: 'An even lighter version of the diff editor colors for contexts when contrast is an issue.',
+            },
+            {
+                id: 'editorLightBulbAutoFix.foreground.light',
+                defaults: {
+                    dark: Color.transparent('editorLightBulbAutoFix.foreground', ONE_HALF_OPACITY),
+                    light: Color.transparent('editorLightBulbAutoFix.foreground', ONE_HALF_OPACITY),
+                },
+                description: 'A lighter version of a blue focused editor state background.',
+            },
+            {
+                id: 'editor.foreground.light',
+                defaults: {
+                    dark: Color.transparent('editor.foreground', ONE_HALF_OPACITY),
+                    light: Color.transparent('editor.foreground', ONE_HALF_OPACITY),
+                },
+                description: 'A lighter version of the text in the editor.',
+            },
+        );
+    }
```

Comment:
We should maybe revisit these colors to see if they are still needed, some of them mention other components such as `diff` and `editor` and perhaps they belong there instead. Else we can rename them to be more meaningful for `memory-inspector`.

Reply:
Renamed.